docs/modding/localisation_files.md

```diff
@@ -1,12 +0,0 @@
-# Localisation
-
-Northstar adds new strings to the game which can be localised to match the language you are using on your Titanfall 2 installation.
-
-## Files to translate
-
-There are the two files that need to be translated, use the English version as a base. After you finish make sure that the files are encoded in `UTF-16 LE` before opening a Pull Request.
-
-1. [Northstar.Client/mod/resource/northstar_client_localisation_english.txt](https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Client/mod/resource/northstar_client_localisation_english.txt)
-2. [Northstar.Custom/mod/resource/northstar_custom_english.txt](https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Custom/mod/resource/northstar_custom_english.txt)
-
-To test your modifications go to `Origin (My games library) -> Titanfall 2 (right click) -> Game Properties -> Advanced Launch Options` and select the language you modified from the dropdown.
```

Comment:
Do we have localisation instructions in the new modding docs? Couldn't find any :c